This patch drops two flags passed to the assembler that forces building for
74xx processors when altivec is enabled, thus breaking compile on G5 (970)
processors.
See bug Gentoo #130217.

Index: xine-lib-1.1.2cvs20060328/configure.ac
===================================================================
--- xine-lib-1.1.2cvs20060328.orig/configure.ac
+++ xine-lib-1.1.2cvs20060328/configure.ac
@@ -1991,8 +1991,6 @@ case "$host_or_hostalias" in
     
     if test x$enable_altivec = xyes; then
         AC_DEFINE_UNQUOTED(ENABLE_ALTIVEC,,[Define this if you have a Motorola 74xx CPU])
-        CFLAGS="-Wa,-m7400 $CFLAGS"
-	DEBUG_CFLAGS="-Wa,-m7400 $DEBUG_CFLAGS"
     fi
     ;;
   sparc*-*-linux*)
